event-management-frontend/src/components/userMapEvents/UserMapEventsDumb.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import { Map, TileLayer } from 'react-leaflet';
+import 'leaflet/dist/leaflet.css';
+import useStylesMapWrapper from '../../styles/MapWrapperStyle';
+import '../../styles/Map.css';
+import UserMapDisplayLocationsDumb from './UserMapDisplayLocationsDumb';
+import { EventCardsWithLoction } from './UserMapEventsSmart';
+import UserMapDisplayChosenLocationDumb from './UserMapDisplayChosenLocationDumb';
+
+interface Props {
+  eventCardsWithLocation: EventCardsWithLoction[];
+}
+
+const UserMapEventsDumb: React.FC<Props> = (props: Props) => {
+  const classesMap = useStylesMapWrapper();
+  const [selectedEvent, setSelectedEvent] = useState<EventCardsWithLoction>({
```

Comment:
Meh. Not exactly happy with this approach. Dumb components shouldn't exactly initialize the object that will be rendered. Instead, this would happen in the smart component. In dumb components you ideally want only local-purpose objects and variables to be used. I.e., not data that comes from other places such as an event object which you want to display, but comes from the smart component. Treat this as a suggestion, not a change request.